### ISSUE TITLE
vkconfig fails when  Vulkan package is not installed on Ubuntu when processing Profile Layer

### ISSUE THREAD

**Issue by davidlunarg** [LunarG/VulkanTools#1639]
The manifest file for the Profile Layer references ${VULKAN_CONTEXT}. On Linux, this expands to a full path from the root dir, which will only exist if the Vulkan package has been installed on the system.

If I have not installed the Vulkan Ubuntu package, or want to work exclusively with an experimental version of the Profile layer inside a build of the Vulkan-Profiles repo, vkconfig gets errors when it tries to process the manifest files for the Profile layer.

**Comment by KarenGhavam-lunarG** [comment]
Why does this need to work correctly outside of the packages being installed?

**Comment by davidlunarg** [comment]
If the tarball sdk is installed but not the packages, vkconfig will try refer to files in folders that only exist if the packages are installed. It will error out.

Developers may want to use vkconfig to configure layers that are in development source repo directories. vkconfig will use files in the installed dirs, not the recently built dev trees.

**Comment by christophe-lunarg** [comment]
I spent a while investigating this issue today and I think it's relevant and question the integration between the Vulkan SDK and the repository and how the manifest setting is designed.

I added an entry to my brainstorm document as I didn't find an easy fix during the investigation:
https://docs.google.com/document/d/1tinO5vNz_mxR26S9Xm_3XFLGmq1RZlInop6eucgaxLU/edit#

So for example, this is path in the profiles manifest: 
"${VULKAN_CONTENT}/VK_LAYER_KHRONOS_profiles/VP_LUNARG_desktop_portability_2021.json"

"${VULKAN_CONTENT}/VK_LAYER_KHRONOS_profiles.json" is clearly a path to the Vulkan SDK. 

This means that:
- VK_LAYER_KHRONOS_profiles.json needs to be packaged with the profiles layer.
- That something need to be done so that we can use the profiles files in the repository when using the profiles layer build from the repository.
